piv/src/main/java/com/yubico/yubikit/piv/jca/PivProvider.java

```diff
@@ -82,7 +82,8 @@ public Object newInstance(Object constructorParameter) {
         try {
             KeyPairGenerator rsaGen = KeyPairGenerator.getInstance("RSA");
             long start = System.currentTimeMillis();
-            for (KeyType keyType : new KeyType[]{KeyType.RSA1024, KeyType.RSA2048}) {
+              for (KeyType keyType : new KeyType[]{KeyType.RSA1024, KeyType.RSA2048, KeyType.RSA3072, KeyType.RSA4096}) {
```

Comment:
Formatting is wrong on this line.

Reply:
My IDE looks correct but here in Github it looks incorrect still

Reply:
looks correct now in gh as well:
![image](https://github.com/Yubico/yubikit-android/assets/1954891/d25b63ff-1f6f-473f-b3fd-338d7bdecbf9)